docs/snippets/web-components/my-component-play-function-composition.ts.mdx

```diff
@@ -34,12 +34,12 @@ export const SecondStory: Story = {
 };
 
 export const CombinedStories: Story = {
-  play: async ({ canvasElement }) => {
-    const canvas = within(canvasElement);
+  play: async (context) => {
```

Comment:
@Foxhoundn can you add a small comment in the example explaining why the user would use `context` instead of the destructured `canvasElement`, so that we can clarify to the readers? Also if you're able, can you also apply this pattern to this [file](https://github.com/storybookjs/storybook/blob/next/docs/snippets/angular/my-component-play-function-composition.ts.mdx), so that folks don't see one example with `canvasElement` and another with `context`